packages/fetchai/agents/aries_alice/aea-config.yaml

```diff
@@ -8,22 +8,24 @@ fingerprint: {}
 fingerprint_ignore_patterns: []
 connections:
 - fetchai/http_client:0.6.0
-- fetchai/oef:0.7.0
+- fetchai/p2p_libp2p:0.6.0
+- fetchai/soef:0.6.0
 - fetchai/stub:0.7.0
 - fetchai/webhook:0.5.0
 contracts: []
 protocols:
 - fetchai/default:0.4.0
-- fetchai/fipa:0.5.0
 - fetchai/http:0.4.0
 - fetchai/oef_search:0.4.0
 skills:
 - fetchai/aries_alice:0.4.0
 - fetchai/error:0.4.0
-default_connection: fetchai/oef:0.7.0
+default_connection: fetchai/p2p_libp2p:0.6.0
 default_ledger: cosmos
 logging_config:
   disable_existing_loggers: false
   version: 1
 private_key_paths: {}
 registry_path: ../packages
+default_routing:
+  fetchai/oef_search:0.4.0: fetchai/soef:0.6.0
```

Comment:
This should also have an entry for http_client

Reply:
Done